testpilot/frontend/static-src/app/views/eol-view.js

```diff
@@ -0,0 +1,18 @@
+import BaseView from './base-view';
+
+export default BaseView.extend({
+  template: `<div class="eol-block"><div data-hook="ending-soon">
+<strong>This experiment is ending on <span data-hook="completedDate"></span></strong>.<br/><br/>
+After then you will still be able to use <span data-hook="title"></span> but we will no longer be providing updates or support.</div>
+<!--<div data-hook="experiment-completed">
+This experiment was retired on <span data-hook="completedDate"></span> and is no longer supported.</br>
+</div>--></div>`,
```

Comment:
I couldn't figure out how to get `data-l10n-id` to work here. The bindings wouldn't fill in like they do with, for example `{$title} is enabled`... Maybe we'll be on to react before we need translations???